packages/protocol/migrations_sol/Migration.s.sol

```diff
@@ -43,6 +43,9 @@ import "@celo-contracts/stability/interfaces/ISortedOracles.sol";
 import "@celo-contracts-8/common/interfaces/IGasPriceMinimumInitializer.sol";
 import "@celo-contracts-8/common/interfaces/IMintGoldScheduleInitializer.sol";
 
+// TODO(Arthur): refactor to use @mento-core remapping. Waiting to merge PR from `master` that has this.
+import "../../lib/mento-core/contracts/ReserveSpenderMultiSig.sol";
```

Comment:
Currently I'm getting a build error here:

```sh
Compiler run failed:
Error (5333): Source file requires different compiler version (current compiler is 0.8.19+commit.7dd6d404.Darwin.appleclang) - note that nightly builds are considered to be strictly less than the released version
 --> lib/mento-core/contracts/ReserveSpenderMultiSig.sol:1:1:
  |
1 | pragma solidity ^0.5.13;
  | ^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
This should be fixed as of https://github.com/celo-org/celo-monorepo/pull/11028/commits/e3fbb84daa183d03f55e5933591b4694007a106c.